### PR TITLE
Fix GPCsvImportTests for non english language settings.

### PR DESCRIPTION
**Pull request by asarbs**

### CODE REVIEW

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java [hunk 1]
```diff
@@ -41,8 +41,7 @@
 import java.io.Reader;
 import java.io.StringReader;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Map;
+import java.util.*;
```
Comment (dbarashev) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: Please get back single imports. We prefer not to use wildcard imports.

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java [hunk 2]
```diff
@@ -104,10 +103,20 @@ public void testImportAssignments() throws Exception {
     HumanResourceManager resourceManager = builder.getResourceManager();
     RoleManager roleManager = new RoleManagerImpl();
 
-    String header1 = "Name,Begin date,End date,Resources,Duration,Completion,Web Link,Notes,Predecessors,ID";
+    String header1 = buildTaskHeader(
+            TaskRecords.TaskFields.NAME,
```
Comment (dbarashev) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: nit: please use 4-spaces for wrapped statements

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java [hunk 3]
```diff
@@ -172,11 +201,44 @@ public void testCustomFields() throws Exception {
     assertEquals("value2", t1.getCustomValues().getValue(def2));
   }
 
+  private String buildTaskHeader(TaskRecords.TaskFields... taskFields) {
+    StringBuilder sb = new StringBuilder();
```
Comment (dbarashev) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: What about  this?
```
Joiner.on(',').join(Stream.of(taskFields).map(TaskRecords.TaskFields::toString).iterator());
```

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java [hunk 4]
```diff
@@ -221,15 +293,20 @@ public void testHierarchy() throws IOException {
...
+            TaskRecords.TaskFields.OUTLINE_NUMBER);
     String data1 = "1,t1,23/07/12,26/07/12,1,1";
     String data2 = "2,t2,23/07/12,24/07/12,1,1.1";
     String data3 = "3,t3,24/07/12,26/07/12,1,1.2";
     String data4 = "4,t4,24/07/12,25/07/12,1,1.2.1";
     String data5 = "5,t5,25/07/12,26/07/12,1,1.2.2";
 
-    GanttCSVOpen importer = new GanttCSVOpen(createSupplier(Joiner.on('\n').join(header1, data1, data2, data3, data4, data5)),
-        taskManager, null, null, builder.getTimeUnitStack());
+    GanttCSVOpen importer = new GanttCSVOpen(createSupplier(Joiner.on('\n').join(header1, data1, data2, data3, data4, data5)), taskManager, null, null, builder.getTimeUnitStack());
```
Comment (dbarashev) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: Please keep the linebreak, long lines are pretty difficult to read.

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java (hunk 1 above)
Comment (asarbs) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: done

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java (hunk 2 above)
Comment (asarbs) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: done

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java (hunk 3 above)
Comment (asarbs) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: Done

File: ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java (hunk 4 above)
Comment (asarbs) [inline ganttproject-tester/test/biz/ganttproject/impex/csv/GPCsvImportTest.java]: Done

**Pull request merged**